components/03-modules/dashboard-toolbar/_dashboard-toolbar.scss

```diff
@@ -0,0 +1,40 @@
+.dashboard-toolbar {
+    &__amount {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        margin-bottom: 15px;
+
+        @include mq($screen-m) {
+            justify-content: flex-start;
+        }
+    }
+
+    &__limiter {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        margin-bottom: 15px;
+
+        @include mq($screen-m) {
+            justify-content: flex-end;
+        }
+    }
+
+    &__pager {
+        margin-bottom: 0;
+    }
+
+    &__information {
+        margin: 0 15px;
+    }
+
+    .select {
```

Comment:
maybe some module class for this select?

Reply:
added